### PR TITLE
Create TensorFunction and LabelAggregationFunction to ensure all models are Send.

### LINKED ISSUE #91: Make all models and pipelines `Send`
Hi @mfelsche,

Over the past few weeks I have worked on making all tokenizers `Send` to make multithreading in the library easier. With these changes, all pipelines and models could be also `Send`. I can see you have worked on a fork to ensure that these pipelines are `Send` for use in one of your projects.

Would you be interested in contributing these changes back to this repository? Changing the `TensorFunction` to a `Struct` and the methods to access the function `get_fn` would be welcome additions (https://github.com/mfelsche/rust-bert/commit/11409893d61afecc76145791d1b59bc4eab5170d and https://github.com/mfelsche/rust-bert/commit/80486069d1eed96eb6438e13c47bb2fdf24003ba)

### PR DESCRIPTION
**Pull request by mfelsche**
I added some ignored tests only for verifying all pipelines and models are indeed `Send`. It is enough if they compile, they don't need to be run actually. If this seems superfluous, i can gladly remove them.

Fixes #91

### CODE REVIEW

**Comment by mfelsche** [comment]
Dunno why but there is 1 test failure in `openai_gpt_lm_model`:

```
thread 'openai_gpt_lm_model' panicked at 'assertion failed: (model_output.lm_logits.double_value(&[0,
                                       model_output.lm_logits.size()[1] - 1,
                                       next_word_id]) - (9.1056)).abs() < 1e-4', tests/openai_gpt.rs:89:5
```

I didnt really touch any of the logic there.

**Comment by mfelsche** [comment]
Turns out i did tough the logic there.

**Review by guillaume-be — APPROVED** [review]
Looking great, thank you for the PR!

File: src/gpt2/transformer.rs [hunk 1]
```diff
@@ -37,17 +35,14 @@ impl MLP {
...
-        let activation = Box::new(match &config.afn {
+        let activation = match &config.afn {
             Some(activation_enum) => match activation_enum {
-                Activation::gelu => _gelu_new,
-                Activation::relu => _relu,
-                Activation::swish => _swish,
-                Activation::gelu_new => _gelu_new,
-                Activation::mish => _mish,
-                Activation::tanh => _tanh,
+                Activation::gelu => &Activation::gelu_new,
```
Comment (mfelsche) [inline src/gpt2/transformer.rs:40]: BTW @guillaume-be this mapping from `gelu` to `gelu_new` is intentional, right?

Comment (guillaume-be) [reply src/gpt2/transformer.rs:40]: Yes - see https://github.com/huggingface/transformers/blob/84caa23301f2f25dfca0737198ff26c3d711ed63/src/transformers/modeling_openai.py#L142

Comment (mfelsche) [reply src/gpt2/transformer.rs:40]: Alright! :) Just wanted to check back as it puzzled me.

**Pull request merged**